### PR TITLE
7383_Levkovich_3

### PR DESCRIPTION
**Pull request by DmLvkvch**

### CODE REVIEW

**Comment by DmLvkvch** [comment]
Здравствуйте, добавил ответы на вопросы

**Comment by Ksenox** [comment]
3) В ответе не понятно, что именно может быть небезопасно для сети
5) Ответ неправильный. Коррекция идет на основе mse, так как в функции compile параметр loss = 'mse'

5,5 баллов
